server/base/src/main/java/org/apache/accumulo/server/tables/TableManager.java

```diff
@@ -115,10 +115,22 @@ public TableState getTableState(String tableId) {
 
     final TableState oldState;
     final TableState newState;
+    final String message;
 
     public IllegalTableTransitionException(TableState oldState, TableState newState) {
+      this(oldState, newState, "");
+    }
+
+    public IllegalTableTransitionException(TableState oldState, TableState newState, String message) {
       this.oldState = oldState;
       this.newState = newState;
+
+      if (message != null && !message.isEmpty())
```

Comment:
Could use StringUtils.isNotEmpty() from apache commons.  Or even static import it so it just reads if(isNotEmpty(message))

Reply:
Thanks, @milleruntime . I like that better.